internal/server/statvar/golden/search_statvar/count_person.json

```diff
@@ -3919,86 +3999,6 @@
     {
       "name": "Population: 1 - 4 Years, Different House in Different County Different State",
       "dcid": "Count_Person_1To4Years_DifferentHouseInDifferentCountyDifferentState"
-    },
```

Comment:
Some of the new variables were added and existing removed from the 1000 SV list.